KrampitzKreutzwiserSettlersOfCatan/src/krampitzkreutzwisersettlersofcatan/GamePanel.form

```diff
@@ -34,17 +34,9 @@
           <Group type="102" attributes="0">
               <EmptySpace max="-2" attributes="0"/>
               <Group type="103" groupAlignment="0" attributes="0">
```

Comment:
In this file, I just removed a couple of Swing elements that I replaced with drawnString from g2d. Just make it easier to scale. Everything should still work the same way though.